custom-ci-cd-environment-with-docker_5caf3f432c7d3a392f9ceb50.md

```diff
@@ -98,7 +105,16 @@ The `hostname` of linked containers is set based on the name of container. In
 the previous example our Postgres database is named `db` and it is available
 on the `db` hostname in the first container.
 
-## Using custom Docker images
+## Pre-built convenience Docker images for Semaphore CI/CD jobs
+
+For convenience, Semaphore comes with a [repository of pre-built images hosted
+on Dockerhub][dockerhub-semaphore]. The source code of the Semaphore Docker
+images is [hosted on Github][docker-images-repo].
+
+- [Android](https://hub.docker.com/r/semaphoreci/android/tags)
+- [Ruby](https://hub.docker.com/r/semaphoreci/ruby/tags)
```

Comment:
Submitted an issue to extend this section https://github.com/semaphoreci/docs/issues/411.